File: praktikum_algopascal/C/AulaNurRizalArdiyantoro_181410045_luasLingkaran.c
//program LuasLingkaran

#include <stdio.h>

//jadikan float
float LL(float r)
{
	float luas;

	luas = 3.14 * r * r;
	//throw when r have n.
	return(luas);
}

int main()
{
	float luas;
	float l;
	printf("Masukan Nilai R: ");
	scanf("%f", &l);

	//throw R into function,  then declare this.
	luas = LL(l);
	printf("Hasil : %f", luas);
	return 0;
}

File: praktikum_algopascal/AulaNurRizalArdiyantoro_181410045_LuasLingkaran.pas
program luasLingkaran;
	uses crt;
		function LL(r: Integer): real;

begin
	LL:= 22/7 * r * r;
end;

	var
		a: Integer;
begin
	writeln('Program Luas Lingkaran');
	write('Masukan Nilai R: ');
	readln(a);
	writeln('=======================');
	write('Hasil Luas Lingkaran: ', LL(a):1:0);
	readln;
end.